cdap-proto/src/main/java/co/cask/cdap/proto/RunRecord.java

```diff
@@ -31,9 +31,13 @@
   @SerializedName("runid")
   private final String pid;
 
-  @SerializedName("start")
+  @SerializedName("starting")
   private final long startTs;
 
+  @SerializedName("start")
+  @Nullable
+  private final Long runTs;
```

Comment:
To ensure that what used to be `startTs` has the same meaning, I've renamed it to `runTs`, and kept the same serialized name.

Then, I added the new `startTs`, which represents the time when the state became `ProgramRunStatus.STARTING`. That will be serialized as `starting`